omero/sysadmins/unix/install-web/install-web-trial.txt

```diff
@@ -86,6 +86,12 @@ will depend on your system, please refer to your web server's manual.
 See :ref:`customizing_your_omero_web_installation`
 for additional customization options.
 
+Set the following:
```

Comment:
As above, you can put the double :: here and move the code extract up to line 91